### PR TITLE
Added “Extra information” and “contactNumber notes” to contact detail

### PR DESCRIPTION
**Pull request by hstandaert**

### CODE REVIEW

File: templates/custom/vesta/vesta.html.twig [hunk 1]
```diff
@@ -54,6 +54,9 @@
               <i class="icon-normalphone" aria-hidden="true"></i>
               <span>
                 <a href="tel:{{ landline.value }}" alt="Person tel">{{ landline.value }}</a>
+                {% if landline.note is not empty %}
```
Comment (meirege) [inline templates/custom/vesta/vesta.html.twig:57]: Not landline.not|render|striptags|trim ?

File: templates/custom/vesta/vesta.html.twig [hunk 2]
```diff
@@ -67,6 +70,9 @@
               <i class="icon-phone" aria-hidden="true"></i>
               <span>
                 <a href="tel:{{ cellular.value }}">{{ cellular.value }}</a>
+                {% if cellular.note is not empty %}
```
Comment (meirege) [inline templates/custom/vesta/vesta.html.twig:73]: Same question as above.

**Review by meirege — CHANGES_REQUESTED** [review]

File: templates/custom/vesta/vesta.html.twig (hunk 1 above)
Comment (hstandaert) [reply templates/custom/vesta/vesta.html.twig:57]: `landline.note` returns a string, as defined in `Gent\Services\Vesta\Value\ContactNumber`, so just checking for an empty string should do 😉

Comment (meirege) [reply templates/custom/vesta/vesta.html.twig:57]: Oh it doesn't pass through Drupal, ok :)

**Review by meirege — APPROVED** [review]

**Pull request merged**